packages/app/src/server/views/widget/page_content.html

```diff
@@ -28,6 +28,7 @@
   data-page-has-children="{% if pages.length > 0 %}true{% else %}false{% endif %}"
   data-page-user="{% if pageUser %}{{ pageUser|json }}{% else %}null{% endif %}"
   data-page-ids-of-seen-users="{{ page.seenUsers|slice(-15)|default([])|reverse|join(',') }}"
+  data-page-ids-of-likers="{{ page.liker|slice(-15)|default([])|reverse|join(',') }}"
```

Comment:
I also think this file is mad, but will be omitted someday when we re-impl with Next.js.